webapp/app/assessment/components/section/assessmentSectionView/assessmentSection.js

```diff
@@ -36,7 +38,9 @@ const AssessmentSection = forwardRef((props, ref) => {
 
       <CustomHeader assessmentType={assessmentType} sectionName={sectionName} disabled={disabled} />
 
-      <Descriptions sectionName={sectionName} descriptions={descriptions} disabled={disabled} />
+      {!isTypePanEuropean(assessmentType) && (
```

Comment:
also GeneralComments should be disabled for pan European

Reply:
Should be handled automatically by:
`descriptions.comments &&`